src/main/resources/assets/purpeille/lang/tn_da.json

```diff
@@ -0,0 +1,119 @@
+  "advancements.purpeille.full_purpeille_set.description": "Ke Keben Imly Purpeillemo Losik",
+  "advancements.purpeille.smoldered_purpeille_ingot.title": "Ke Keben Ma Lymissis",
+  "advancements.purpeille.smoldered_purpeille_ingot.description": "Ke saoRen Familath Purpeillemo Ikasibef",
+  "advancements.purpeille.warpath.title": "Eaf Kastia Myl",
+  "advancements.purpeille.warpath.description": "Ke Nanli Purpeille Warpathsis",
+  "advancements.purpeille.warpath_terror.title": "Ma Stasalareth Yalo",
+  "advancements.purpeille.warpath_terror.description": "Ke Lam Amas Nalthir Purpeillemo",
+  "advancements.purpeille.warpath_opposition.title": "Akmo Stymo",
+  "advancements.purpeille.warpath_opposition.description": "Ke Farasat Aurmalo saoStasai",
+  "advancements.purpeille.warpath_pace.title": "Lokta Ramalamo",
+  "advancements.purpeille.warpath_pace.description": "Faras Stasai Jithaka",
+  "advancements.purpeille.ancient_mechanical_bricks.title": "Saoasa Balo Dofthen Thumoko",
+  "advancements.purpeille.ancient_mechanical_bricks.description": "Ke Keben Kekafanota Asala Basakys",
+  "advancements.purpeille.ancient_oven.title": "Lok Stym Fymimo",
+  "advancements.purpeille.ancient_oven.description": "Ke Keben Jaym Asala",
+  "advancements.purpeille.repair_ancient_oven.title": "Lam Jama",
+  "advancements.purpeille.repair_ancient_oven.description": "Ke Nymus Lanom Jaym Asala",
+  "advancements.purpeille.encased_core.title": "Ke Feithsa Ma",
+  "advancements.purpeille.encased_core.description": "Ke Ren Ramala Jamakys Jaymo",
+  "advancements.purpeille.vacuous_core.title": "Ke Ren saoFaula",
+  "advancements.purpeille.vacuous_core.description": "Ijym Void Ren Ramala Jamakys",
+  "advancements.purpeille.overclocked_core.title": "Ma Sen Demonal",
+  "advancements.purpeille.overclocked_core.description": "Ke Faras Ramala Akika Ia",
+  "advancements.purpeille.ancient_gateway.title": "Jis Losyj",
+  "advancements.purpeille.ancient_gateway.description": "Isai Dama Asala",
+  "advancements.purpeille.ancient_gateway_far.title": "Basaka Voido",
+  "advancements.purpeille.ancient_gateway_far.description": "Basa 10,000 Fantaota Dama Asalasis",
+  "command.purpeille.warpath.invalid.revelation": "Revelation Aki ul %s lu",
+  "command.purpeille.warpath.invalid.aspect": "Aspect Aki ul %s lu",
+  "command.purpeille.warpath.invalid.stack": "%s saoKaset ma",
+  "command.purpeille.warpath.success.add": "Faras Warpath ul [%s] lu",
+  "command.purpeille.warpath.success.remove": "Kekath Warpath ul [%s] lu",
+  "subtitle.purpeille.ancient_gateway.teleport": "Dama Asala Doften",
+  "use.purpeille.creative_core": "Farasta Ramala %dsa",
+  "tooltip.purpeille.creative_core": "Ramala Sa: ",
+  "rei.purpeille.ancient_oven": "Faras Voidsis",
+  "rei.purpeille.ancient_oven.damage": "Kekath %d",
+  "patchouli.purpeille.ancients_almanac.landing_text": "Ke Losa $(italic)tanoIlliaf Mamu$() na Feithsat Ke ma Jasi befTano!"
```

Comment:
oops I didn't expect you to translate this, the only translation of the guidebook is in english atm so you should probably remove this to keep it consistent for now